### PR TITLE
[events] remove message field

### PR DESCRIPTION
**Pull request by alangenfeld**
we have been double writing the message for user logs forever 

## Test Plan

bk, verify experience same in dagit

### CODE REVIEW

**Comment by alangenfeld** [comment]
Current dependencies on/for this PR:
* master
  * **PR #6769** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/6769" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: python_modules/dagster-graphql/dagster_graphql/implementation/events.py [hunk 1]
```diff
@@ -362,9 +362,7 @@ def construct_basic_params(event_record):
     check.inst_param(event_record, "event_record", EventLogEntry)
     return {
         "runId": event_record.run_id,
-        "message": event_record.dagster_event.message
-        if (event_record.dagster_event and event_record.dagster_event.message)
-        else event_record.user_message,
```
Comment (alangenfeld) [inline python_modules/dagster-graphql/dagster_graphql/implementation/events.py:367]: this behavior is what justifies the diff

File: python_modules/dagster/dagster/core/events/log.py [hunk 2]
```diff
@@ -124,12 +118,23 @@ def from_json(json_str):
...
+        """
+        Return the message from the structured DagsterEvent if present, fallback to user_message
+        """
+
+        if self.is_dagster_event:
+            msg = self.get_dagster_event().message
+            if msg is not None:
+                return msg
+
+        return self.user_message
```
Comment (alangenfeld) [inline python_modules/dagster/dagster/core/events/log.py:131]: replicated here

Comment (prha) [reply python_modules/dagster/dagster/core/events/log.py:131]: prefer the signature change rather than adding a property?

**Review by prha — APPROVED** [review]

File: python_modules/dagster/dagster/core/events/log.py (hunk 2 above)
Comment (alangenfeld) [reply python_modules/dagster/dagster/core/events/log.py:131]: not a bad idea - would reduces changes required

**Pull request merged**